lib/rbac/filterer.rb

```diff
@@ -319,6 +321,8 @@ def search(options = {})
       attrs[:auth_count] = auth_count unless options[:skip_counts]
 
       return targets, attrs
+    rescue ActiveRecord::EagerLoadPolymorphicError
+      raise Rbac::PolymorphicError
```

Comment:
Does the exception message include the polymorphic that was included?

Reply:
This is just a nice to have... hopefully if it blows up, it will tell you which polymorphic is at fault.